test/core/test_substitutions.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+import logging
+
+# ==================================================================================
+
+from crds.core import log
+from crds.core import substitutions
+
+# ==================================================================================
+
+log.THE_LOGGER.logger.propagate=True
+log.set_verbose(50)
+
+@pytest.mark.substitutions
```

Comment:
So in order to add the mark "substitutions", you need to add it to pyproject.toml?

Reply:
Yes, all markers have to be added there